pkg/pfcpsim/pfcpsim.go

```diff
@@ -166,16 +166,12 @@ func (c *PFCPClient) sendMsg(msg message.Message) error {
 	return nil
 }
 
-func (c *PFCPClient) receiveFromN4(ctx context.Context) {
+func (c *PFCPClient) receiveFromN4() {
 	buf := make([]byte, 3000)
 
 	for {
 		select {
-		case <-ctx.Done():
-			if c.cancelHeartbeats != nil {
-				c.cancelHeartbeats()
```

Comment:
Does the `cancelHeartbeats()` need to be handle in a different way? or there is no need to handle it?

Reply:
We don't need to handle it, because the done event is triggered by `c.cancelHeartbeats()`.